### PR TITLE
Fill trigger action for backend forms

### PR DESCRIPTION
**Pull request by JLyons009**
Added and compiled js that will allow 'fill' trigger action for backend forms.  Use case is 'select all' in a backend form.

### CODE REVIEW

**Comment by LukeTowers** [comment]
@JLyons009 we'll need this added to the documentation: https://octobercms.com/docs/backend/forms#field-trigger-events and https://octobercms.com/docs/ui/input-trigger

File: modules/system/assets/ui/storm-min.js [hunk 1]
```diff
@@ -3296,7 +3296,7 @@ FilterWidget.prototype.getPopoverRangeTemplate=function(){return'
                                         type="text"                                                                       \
                                         name="date"                                                                       \
                                         value="{{ date }}"                                                                \
-                                        class="form-control align-right"                                                  \
+                                        class="form-control align-right popup-allow-focus"                                                  \
```
Comment (w20k) [inline modules/system/assets/ui/storm-min.js]: @JLyons009 how do you get additional class `popup-allow-focus` added after re-compiling js and no adding it manually...I'm lost. Need to check!

Comment (JLyons009) [reply modules/system/assets/ui/storm-min.js]: Not sure what caused that addition, but I resolved it!

**Comment by JLyons009** [comment]
I'm not sure why the checks failed, can I get some assistance on how to correct?  Thanks!

**Comment by w20k** [comment]
@JLyons009 no worries, it's not related to your PR. Will be fixed soon!

File: modules/system/assets/ui/js/input.trigger.js [hunk 2]
```diff
@@ -140,6 +140,19 @@
...
+            this.$el
+                .not('input[type=checkbox], input[type=radio], input[type=button], input[type=submit]')
+                .val('')
+
+            this.$el
+                .not(':not(input[type=checkbox], input[type=radio])')
+                .prop('checked', true)
+
+            this.$el
+                .trigger('empty.oc.triggerapi')
```
Comment (LukeTowers) [inline modules/system/assets/ui/js/input.trigger.js]: This event should be `fill.oc.triggerapi`.

File: modules/system/assets/ui/js/input.trigger.js [hunk 3]
```diff
@@ -140,6 +140,19 @@
                 .trigger('empty.oc.triggerapi')
                 .trigger('change')
         }
+        else if (action == 'fill' && status) {
+            this.$el
+                .not('input[type=checkbox], input[type=radio], input[type=button], input[type=submit]')
+                .val('')
```
Comment (LukeTowers) [inline modules/system/assets/ui/js/input.trigger.js]: What is this doing? Looks like copy paste from the empty action

Comment (w20k) [reply modules/system/assets/ui/js/input.trigger.js]: @JLyons009 could you merge both actions (empty & fill) in one and just with ternarri operator use it for action|(true/false checkbox)

Comment (JLyons009) [reply modules/system/assets/ui/js/input.trigger.js]: Done.  Let me know if any more revisions are needed.  Thanks!

**Comment by w20k** [comment]
@JLyons009 could you re-build the assets as well?

**Comment by w20k** [comment]
@JLyons009 could you update docs? Will then re-test. But will be merged only in the next release.
This PR is pretty much `completed`, needs only docs update.

**Comment by w20k** [comment]
docs: https://github.com/octobercms/docs/pull/361

**Comment by daftspunk** [comment]
The use case for this is not completely described:

> Use case is 'select all' in a backend form.

Please provide the usage example code

**Pull request closed without merging**